### PR TITLE
[USETUP] Polish translation update

### PR DESCRIPTION
**Pull request by pithwz**
[usetup] [pl-PL] update

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-polish

File: base/setup/usetup/lang/pl-PL.h [hunk 1]
```diff
@@ -1815,19 +1816,22 @@ static MUI_ENTRY plPLRegistryEntries[] =
         4,
         3,
         " Instalator ReactOS " KERNEL_VERSION_STR " ",
-        TEXT_STYLE_UNDERLINE
+        TEXT_STYLE_UNDERLINE,
+        TEXT_ID_STATIC
```
Comment (binarymaster) [inline base/setup/usetup/lang/pl-PL.h:1820]: Do you keep these style changes in sync with other translations?

Comment (pithwz) [reply base/setup/usetup/lang/pl-PL.h:1820]: yes, these lines are missing in [pl-PL]. Compared 2 files with Winmerge ... ;)

**Review by Saibamen — APPROVED** [review]

File: base/setup/usetup/lang/pl-PL.h [hunk 2]
```diff
@@ -1546,7 +1547,7 @@ static MUI_ENTRY plPLBootLoaderInstallPageEntries[] =
     {
         0,
         0,
-        "Installing the bootloader onto the media, please wait...",
+        "Instalacja mened\276era rozruchu na nosniku, Prosz\251 czeka\206...",
```
Comment (archeYR) [inline base/setup/usetup/lang/pl-PL.h:1550]:
```suggestion
        "Instalacja mened\276era rozruchu na nosniku, prosz\251 czeka\206...",
```

**Review by archeYR — CHANGES_REQUESTED** [review]

**Review by archeYR — APPROVED** [review]

**Pull request merged**